dashboard/app/models/sections/section.rb

```diff
@@ -578,7 +568,36 @@ def update_code_review_expiration(enable_code_review)
   end
   before_validation :strip_emoji_from_name
 
-  public def add_instructor(email, current_user)
+  # Adds an instructor to the section
+  # If the instructor was previously deleted, restore the instructor
+  # If the instructor was invited or inactive, reactive the instructor
+  # If the instructor did not previously exist, create the section instructor relationship
+  # Returns true if the instructor could be added, false otherwise
+  public def add_instructor(user)
```

Comment:
In Ruby, if a method is not under `private` or `protected` blocks, it's `public` by default.
```suggestion
  def add_instructor(user)
```

Reply:
Thanks! I was following the convention as others were also using `public` in the file but I updated it.